x-pack/plugins/watcher/__jest__/client_integration/watch_edit.test.ts

```diff
@@ -24,12 +24,7 @@ jest.mock('../../public/application/lib/api', () => {
   return {
     ...original,
     loadIndexPatterns: async () => {
-      const INDEX_PATTERNS = [
-        { attributes: { title: 'index1' } },
-        { attributes: { title: 'index2' } },
-        { attributes: { title: 'index3' } },
-      ];
-      return await INDEX_PATTERNS;
+      return ['index1', 'index2', 'index3'];
```

Comment:
nit:  could you please check tests which use that mock? I see `loadIndexPatterns` returns `Promise<SendRequestResponse>` which is not matching to `Promise<string[]>`

At least it should be 
```
  loadIndexPatterns: async () => {
      return { data: ['index1', 'index2', 'index3'] };
    },
```

But my main question is why the tests work =)

Reply:
I just double checked and seems they are not being used anywhere, which explains why they were passing anyway. I'll go ahead and get rid of them :)